dashboard/apps/prototype/static/prototype/main.css

```diff
@@ -0,0 +1,31 @@
+
+
+.login_column {
```

Comment:
i think the convention for css classes is to use `-` instead of `_` for css class names.

https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Underscores_in_class_and_ID_Names